UXPin/adele
# Remove duo security DS data

Duo Security has asked to remove their DS as it is no longer publicly available.

More info here: https://github.com/UXPin/adele/issues/155

**How to test**
1. checkout to this branch: `remove-ds-entry`
2. `npm run data-build`
3. `npm run serve`
4. search by company: Duo Security. If there entry is no longer there then it was successful
